zinc/src/main/scala/sbt/internal/inc/MixedAnalyzingCompiler.scala

```diff
@@ -496,15 +497,51 @@ object MixedAnalyzingCompiler {
 
   /**
    * Create a an analysis store cache at the desired location.
-   *
-   * Note: This method will be deprecated after Zinc 1.1.
    */
-  def staticCachedStore(analysisFile: Path, useTextAnalysis: Boolean): AnalysisStore = {
-    import xsbti.compile.AnalysisStore
-    val fileStore =
-      if (useTextAnalysis) sbt.internal.inc.FileAnalysisStore.text(analysisFile.toFile)
-      else sbt.internal.inc.FileAnalysisStore.binary(analysisFile.toFile)
-    val cachedStore = AnalysisStore.getCachedStore(fileStore)
-    staticCache(analysisFile, AnalysisStore.getThreadSafeStore(cachedStore))
+  def staticCachedStore(analysisFile: Path, useTextAnalysis: Boolean): XAnalysisStore =
+    staticCachedStore(
+      analysisFile = analysisFile,
+      useTextAnalysis = useTextAnalysis,
+      useConsistent = false,
+      mappers = ReadWriteMappers.getEmptyMappers(),
+      sort = true,
+      ec = ExecutionContext.global,
+      parallelism = Runtime.getRuntime.availableProcessors(),
+    )
+
+  def staticCachedStore(
+      analysisFile: Path,
+      useTextAnalysis: Boolean,
+      useConsistent: Boolean,
+      mappers: ReadWriteMappers,
+      sort: Boolean,
+      ec: ExecutionContext,
+      parallelism: Int,
+  ): XAnalysisStore = {
+    import sbt.internal.inc.{ FileAnalysisStore => Store }
```

Comment:
Here as well, I think using `FileAnalysisStore` directly is clearer than the alias `Store`, which is quite ambiguous.